parsely/build.gradle

```diff
@@ -27,27 +29,38 @@ android {
             withJavadocJar()
         }
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
+
+    koverReport {
+        filters {
+            excludes {
+                classes(
+                        "*.BuildConfig"
+                )
+            }
+        }
+    }
 }
 
 dependencies {
-    def kotlinVersion = "1.8.10"
-
     implementation 'androidx.appcompat:appcompat:1.4.2'
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.3'
     implementation 'com.google.android.gms:play-services-ads-identifier:18.0.1'
     implementation 'androidx.lifecycle:lifecycle-process:2.6.2'
-    constraints {
-        add("implementation", "org.jetbrains.kotlin:kotlin-stdlib-jdk7") {
-            version {
-                require(kotlinVersion)
-            }
-        }
-        add("implementation", "org.jetbrains.kotlin:kotlin-stdlib-jdk8") {
-            version {
-                require(kotlinVersion)
-            }
-        }
-    }
+    
+    testImplementation 'org.robolectric:robolectric:4.9.2'
```

Comment:
Suggestion (💡): Consider using the latest version of Robolectric ([4.10.3](https://github.com/robolectric/robolectric/releases/tag/robolectric-4.10.3)).